tripal_chado/tests/src/Functional/api/ChadoOrganismAPITest.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+namespace Drupal\Tests\tripal_chado;
+
+use Drupal\Core\Url;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Core\Database\Database;
+use Drupal\tripal_chado\api\ChadoSchema;
+
+/**
+ * Testing the tripal_chado/api/tripal_chado.organism.api.inc functions.
+ *
+ * @group Tripal
+ * @group Tripal Chado
+ * @group Tripal API
+ */
+class ChadoOrganismAPITest extends BrowserTestBase {
+
+  protected $defaultTheme = 'stable';
+
+  /**
+   * Modules to enable.
+   * @var array
+   */
+  protected static $modules = ['tripal', 'tripal_chado'];
+
+  /**
+   * Schema to do testing out of.
+   * @var string
+   */
+  protected static $schemaName = 'testchado';
+
+  /**
+   * Tests the following organism API functions:
+   *   chado_get_organism()
+   *   chado_get_organism_scientific_name()
+   *   chado_get_organism_select_options()
+   *   chado_abbreviate_infraspecific_rank()
+   *   chado_unabbreviate_infraspecific_rank()
+   *
+   * @group tripal-chado
+   * @group chado-organism
+   */
+  public function test_chado_organism_api_functions() {
+
+    putenv('TRIPAL_SUPPRESS_ERRORS=TRUE');
+
+    // Lookup cvterm_id for 'subspecies'
+    $cvterm = chado_get_cvterm(['name' => 'subspecies'], [], self::$schemaName);
+    $this->assertNotNull($cvterm, 'test_chado_organism_api_functions() unable to retrieve cvterm for subspecies');
```

Comment:
The failed test result already tells you the test function in the stack trace so I like to make this message specific to the part of the test I'm currently running. For example,

```suggestion
    $this->assertNotNull($cvterm, 'Unable to retrieve cvterm for subspecies using chado_get_cvterm when testing the Organism API.');
```

But this is just a preference change and yours is good too :-) So feel free to ignore.